daemon/src/monitor.rs

```diff
@@ -708,14 +815,17 @@ impl xtra::Message for Sync {
 #[async_trait]
 impl<C> xtra::Actor for Actor<C>
 where
-    C: Send + 'static,
+    C: Send + 'static + bdk::electrum_client::ElectrumApi,
     Self: xtra::Handler<Sync>,
 {
     async fn started(&mut self, ctx: &mut xtra::Context<Self>) {
         let fut = ctx
             .notify_interval(Duration::from_secs(20), || Sync)
             .expect("we just started");
 
+        let addr = ctx.address().expect("Able to acquire own address");
+        let _ = self.rebroadcast_transactions(&addr).await;
```

Comment:
Don't do this. The call to `send` within `rebroadcast` will deadlock this because we are not started yet.

Can we just do this as normal function calls within the constructor?